bin/moonpick

```diff
@@ -20,45 +20,79 @@ Usage: moonpick [opts] <file>, [file2, ..]
 
 Where [opts] can be the following:
 
--v Shows moonpick version
--h Shows this help
+-v                Shows moonpick version
+-h                Shows this help
+--filename file   Filename used for lint_config with stdin
```

Comment:
Yeah the --filename flag is used with stdin, so when linting it knows which linting rules to use. If you have a better name for the flag let me know.